firestore-bigquery-export/README.md

```diff
@@ -126,6 +126,8 @@ To install an extension, your project must be on the [Blaze (pay as you go) plan
 
 * Collection path: What is the path of the collection that you would like to export? You may use `{wildcard}` notation to match a subcollection of all documents in a collection (for example: `chatrooms/{chatid}/posts`). Parent Firestore Document IDs from `{wildcards}`  can be returned in `path_params` as a JSON formatted string.
 
+* Enable logging failed exports: If enabled, the extension will exports that failed to enqueue to the Firebase console, to prevent data loss.
```

Comment:
```suggestion
* Enable logging failed exports: If enabled, the extension will export what failed to enqueue to the Firebase console, to prevent data loss.
```